pkg/registries/types/types.go

```diff
@@ -6,6 +6,38 @@ import (
 	"github.com/stackrox/rox/generated/storage"
 )
 
+const (
+	// ArtifactType represents the Google artifact image integration.
+	ArtifactType = "artifactregistry"
+
+	// ArtifactoryType represents the Artifactory image integration.
+	ArtifactoryType = "artifactory"
+
+	// AzureType represents the Azure container registry image integration.
+	AzureType = "azure"
+
+	// DockerType represents the docker image integration.
+	DockerType = "docker"
+
+	// ECRType represents the AWS ECR image integration.
+	ECRType = "ecr"
+
+	// GoogleType represents the Google container registry image integration.
+	GoogleType = "google"
+
+	// IBMType represents the IBM image integration.
+	IBMType = "ibm"
+
+	// NexusType represents the Nexus image integration.
+	NexusType = "nexus"
+
+	// QuayType represents the Quay image integration.
+	QuayType = "quay"
+
+	// RHELType represents the RHEL image integration.
```

Comment:
I think this is really just Red Hat. Not sure why we called it RHEL in the first place. Let's change this to `RedHatType` at least